CondFormats/JetMETObjects/data/Summer15_V5_MC_JER_SF_AK4PFchs.txt

```diff
@@ -0,0 +1,8 @@
+{1    JetAbsEta  0     None      Resolution} 
+0     0.5     3     1.079     1.053      1.105      
+0.5   1.1     3     1.099     1.071      1.127      
+1.1   1.7     3     1.121     1.092      1.150      
+1.7   2.3     3     1.208     1.162      1.254      
+2.3   2.8     3     1.254     1.192      1.316      
+2.8   3.2     3     1.395     1.332      1.458      
+3.2   5.0     3     1.056     0.865      1.247      
```

Comment:
you should remove this from here and add it to the cms-data repo, and change the tests accordingly by using `edm::FileInPath`.